pact4s/README.md

```diff
@@ -0,0 +1,3 @@
+# pact4s [Under construction]
+
+pact4s is used for contract testing.
```

Comment:
I will fill this out a bit more in the next PR. In particular, the command to create the pact contract file is:

`sbt "project pact4s" test`